app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -108,6 +111,7 @@ def check_enrollments(enrollments)
       rescue Faraday::BadRequestError => err
         handle_bad_request_error(err, enrollment)
       rescue StandardError => err
+        NewRelic::Agent.notice_error(err)
```

Comment:
These errors are unexpected, so I'm explicitly passing them to NewRelic so we can investigate them.